File: 04 소프티어/Level 1/Java/src/Prob6253.java

/**
* title  : 6253. 주행거리 비교하기 (Level 1)
* time   : 66ms
* memory : 10.11MB
*/

import java.io.*;
import java.util.*;

public class Prob6253 {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int A = Integer.parseInt(st.nextToken());
        int B = Integer.parseInt(st.nextToken());

        if (A == B) {
            System.out.println("same");
        } else if (A < B) {
            System.out.println("B");
        } else if (A > B) {
            System.out.println("A");
        }
        
    }
    
}

File: 04 소프티어/Level 1/Python/Prob6253.py

"""
title  : 6253. 주행거리 비교하기 (Level 1)
time   : 106ms
memory : 37.32MB
"""

import sys

def solution():
    input = sys.stdin.readline

    A, B = map(int, input().split())

    if A > B:
        print("A")
    elif A < B:
        print("B")
    else:
        print("same")

solution()